client/src/modules/list/model/actions.js

```diff
@@ -115,6 +115,10 @@ const restoreListRequest = () => ({
   type: ListActionTypes.RESTORE_REQUEST
 });
 
+export const removeArchivedListsMetaData = () => ({
+  type: ListActionTypes.REMOVE_ARCHIVED_META_DATA
+});
+
```

Comment:
should here be success, request and failure??

Reply:
Archived lists are removing from Redux only. We don't send a request to the server in this case. So I think there is no need to create failure, success and request actions.